### PR TITLE
Fix canMakePayments

### PR DESCRIPTION
**Pull request by wsliaw**

### CODE REVIEW

File: iphone/Classes/TiApplepayModule.m [hunk 1]
```diff
@@ -67,6 +67,8 @@ - (NSNumber *)isSupported:(id)unused
 
 - (NSNumber *)canMakePayments:(id)args
 {
+    args = [args objectAtIndex:0];
```
Comment (hansemannn) [inline iphone/Classes/TiApplepayModule.m]: Can you use `ENSURE_SINGLE_ARG(args, NSDictionary);` instead and ping back if it works? 😙

**Review by hansemannn — CHANGES_REQUESTED** [review]
Great catch! Small change, then we should be good to go. Please also bump the manifest version to `1.5.1`, thx!

File: iphone/Classes/TiApplepayModule.m (hunk 1 above)
Comment (wsliaw) [reply iphone/Classes/TiApplepayModule.m]: Yes it works

**Review by hansemannn — APPROVED** [review]

**Comment by hansemannn** [comment]
Release published! Btw, now that Titanium supports dynamic frameworks / Swift frameworks, we can add more payment-gateways easily! I'll spend some time on that soon.

**Pull request merged**